setup.py

```diff
@@ -34,7 +34,7 @@ def is_ubuntu() -> bool:
 
 install_requires = [
     "PyYAML",
-    "progressbar",
+    "pychroot==0.10.3",  # 0.10.4 drops Python 3.7 support
```

Comment:
What happened to `progressbar`?

Reply:
Progress display was dropped a few releases ago but the package dependency was not removed.